src/main/java/reposense/RepoSense.java

```diff
@@ -27,6 +27,9 @@
 import reposense.util.FileUtil;
 
 public class RepoSense {
+    public static final String VERSION = getVersion();
```

Comment:
How about getting the version with the static method instead of static variable? I feel this is quite weird, unless there is reason behind it.